docs/source/recipes/librispeech/tdnn_lstm_ctc.rst

```diff
@@ -224,7 +224,7 @@ After downloading, you will have the following files:
       |       `-- G_4_gram.pt
       |-- exp
       |   `-- pretrained.pt
-      `-- test_wavs
+      `-- test_waves
```

Comment:
This one is correct. Please see
https://huggingface.co/pkufool/icefall_asr_librispeech_tdnn-lstm_ctc/tree/main/test_wavs